app/routes/users/components/GroupChange.js

```diff
@@ -75,7 +75,11 @@ class GroupChange extends Component<Props, State> {
           isClearable={false}
         />
         {this.state.selectedOption && (
-          <Button className={styles.saveButton} onClick={this.handleOnClick}>
+          <Button
+            className={styles.saveButton}
```

Comment:
This also only adds margin-top, and that is the only thing in that entire file :D Maybe we can remove that

Reply:
A quick `git blame`: _Peder Smith, 4 years ago via PR #1255_

Reply:
😡